index.js

```diff
@@ -1,3 +1,11 @@
-/* for module loading using webpack or similar package bundlers */
-window.Flow = require('./dist/ng-flow-standalone');
-module.exports = 'flow';
+/* backwards compatable with deprecated index.js*/
+try {
+  console.warn(new Error("use const ngFlow = require('@flowjs/ng-flow');"));
+} catch (e) {
+  // do nothing
+}
+
+if (typeof window === 'object') {
+  window.Flow = require('@flowjs/flow.js');
```

Comment:
mutating window in a commonjs module is not acceptable, but I do it here for require('@flowjs/ng-flow/index') for backwards compatibility reasons.